src/main/java/mate/academy/spring/MainApp.java

```diff
@@ -41,6 +52,32 @@ public static void main(String[] args) throws SQLException {
             System.out.println("Email = " + user.getEmail());
             System.out.println();
         }
+        List<Author> foundedAuthors = authorService.findByName("Taras");
+        for (Author author: foundedAuthors) {
+            System.out.println("Id = " + author.getAuthorId());
+            System.out.println("First Name = " + author.getName());
+            System.out.println("Last Name = " + author.getSurname());
+            System.out.println();
+        }
+
+        authorService.findByNameAndSurname("Taras", "Shevchenko");
```

Comment:
```suggestion
        Author tarasShevchenko = authorService.findByNameAndSurname("Taras", "Shevchenko");
```
@TarasHryts print this author info, and print the books from this author

Reply:
done